leshan-server-cf/src/main/java/org/eclipse/leshan/server/californium/impl/LeshanServer.java

```diff
@@ -87,6 +87,8 @@
 
     private final LwM2mRequestSender requestSender;
 
+    private final LwM2mRequestSender queueModeRequestSender;
```

Comment:
I propose to have only 1 LwM2mRequestSender attribute in LeshanServer.

We keep the 2 implementations QueuemodeSender and "CaliforniumSender".

QueueModeSender will do something like this:

```java
if (!registration.usesQueueMode()) {
     delegate.send(....);
} else {
    // DO queueu mode sutff
```